gateway-incoming.yaml

```diff
@@ -95,6 +95,7 @@ paths:
               "status":
               #! if eq .Response.Body.errorType "AssetNotFoundError" !# 404,
               #! else if eq .Response.Body.errorType "AssetInventoryRequestError" !# 502,
+              #! else if eq .Response.Body.errorType "AssetInventoryMultipleAttributionErrors" !# 404,
```

Comment:
Is this really supposed to be 404?  Seems like something in the 5xx range would be more appropriate.

Reply:
maybe like a 502?